pyproject.toml

```diff
@@ -17,6 +17,24 @@ known_third_party = [
     "sktime",
     "statsmodels",
     "tqdm",
+    "Pillow",
+    "boto3",
+    "requests",
+    "timm",
+    "torch",
+    "torchvision",
+    "torchtext",
+    "fairscale",
+    "scikit-image",
+    "smart_open",
+    "pytorch_lightning",
+    "torchmetrics",
+    "transformers",
+    "nptyping",
+    "omegaconf",
+    "pytorch-metric-learning",
+    "nlpaug",
+    "nltk",
```

Comment:
@yinweisu Is it safe to add these new dependencies?

Reply:
Not too familiar with it, but seems it's only used by isort. So should be fine